skietbaan_fe/src/components/Posts.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux';
+import { fetchPosts } from '../actions/postActions'
+
+class Posts extends Component {
+
+  // a constructor that maintanes the state of the component
+ /**  constructor(props ){
```

Comment:
So this is Ghost Code,  although it has a comment _ a constructor that maintanes the state of the component_ you still need to add a *TODO: why is it there?*

Reply:
Sorted. Kindly review